frontend/src/metabase/components/ConfirmContent.tsx

```diff
@@ -6,7 +6,20 @@ import ModalContent from "metabase/components/ModalContent";
 
 import Button from "metabase/core/components/Button";
 
-const nop = () => {};
+const nop = () => {
+  // Do nothing
+};
+
+interface ConfirmContentProps {
```

Comment:
Converting to TypeScript is necessary because it's used in `<InlineActionSettings />` and TypeScript start inferring obscenely wrong types like `content: null` 🤦